cypress/e2e/demoqa.cy.js

```diff
@@ -0,0 +1,40 @@
+describe('demoQA - Registion Form', () => {
+  const testInput = {
+    firstName: 'Nghia',
+    lastName: 'Do',
+    email: 'meomeo211@gmail.com',
+    gender: 'Female',
+    phoneNumber: '0774859984',
+    //DOB: '14/11/2001',
```

Comment:
You can leave DoB here, then split in to an array to get the value of day, month, year
Ref: [JS split()](https://www.w3schools.com/jsref/jsref_split.asp)